### DISCUSSION TITLE
Where are the Discovery Kit and Video Kit U-Boot DTS files?

### DISCUSSION THREAD

**Discussion by pcrosemurgy** [polarfire-soc/polarfire-soc-documentation#518] [HSS and U-Boot] [answered]
The U-Boot repo `linux4microchip/u-boot-mchp` contains these `.dts` files: https://github.com/linux4microchip/u-boot-mchp/tree/master/arch/riscv/dts
```
microchip-mpfs-icicle-kit-u-boot.dtsi
microchip-mpfs-icicle-kit.dts
microchip-mpfs.dtsi
```

There are no Discovery Kit or Video Kit U-Boot DTS files though. I see that these exist in the old repo (`polarfire-soc/u-boot`, see https://github.com/polarfire-soc/u-boot/tree/mpfs-uboot-2023.07/arch/riscv/dts). Are there plans to move these DTS files to the new U-Boot repo? Or do the DTS files for these boards exist elsewhere?

Thanks!

**Comment by vfalanis** [comment, marked as answer]
Hi @pcrosemurgy ,

The latest PolarFire SoC Linux release is using the u-boot-2023.07-mchp branch. You can find the Discovery Kit and Video Kit dts files there. Please refer to the following link:

https://github.com/linux4microchip/u-boot-mchp/tree/u-boot-2023.07-mchp/arch/riscv/dts
